### PR TITLE
Fix wrong commends in test-cmd.sh

### LINKED ISSUE #18800: Some Comments in test-cmd.sh are wrong
Tests for pods in ```test-cmd.sh``` just check whether pods are exist or not, they don't check the status of pods. So I think the comments as follows in ```test-cmd.sh```  are wrong
~~~
# Pre-condition: no POD is running
# Post-condition: valid-pod POD is running
~~~
Maybe could be replaced with the follows
~~~
# Pre-condition: no POD is exist
# Post-condition: valid-pod POD is created
~~~

### PR DESCRIPTION
**Pull request by feihujiang**
Closes #18800

### CODE REVIEW

File: hack/test-cmd.sh [hunk 1]
```diff
@@ -267,11 +267,11 @@ runTests() {
   kube::log::status "Testing kubectl(${version}:pods)"
 
   ### Create POD valid-pod from JSON
-  # Pre-condition: no POD is running
+  # Pre-condition: no POD is exist
```
Comment (eparis) [inline hack/test-cmd.sh]: I believe the correct phrase would be `no POD exists`.

File: hack/test-cmd.sh [hunk 2]
```diff
@@ -291,117 +291,117 @@ runTests() {
   output_pod=$(kubectl get pod valid-pod -o yaml --output-version=v1 "${kube_flags[@]}")
 
   ### Delete POD valid-pod by id
-  # Pre-condition: valid-pod POD is running
+  # Pre-condition: valid-pod POD is exist
```
Comment (eparis) [inline hack/test-cmd.sh]: And here would be `valid-pod POD does not exist`

**Comment by feihujiang** [comment]
@k8s-bot test this please

**Comment by feihujiang** [comment]
@k8s-bot test this please

**Comment by feihujiang** [comment]
@k8s-bot test this please

**Comment by feihujiang** [comment]
@eparis Addressed the comments, PTAL, thanks.

**Comment by feihujiang** [comment]
@k8s-bot test this please

**Comment by feihujiang** [comment]
FAIL	k8s.io/kubernetes/plugin/pkg/scheduler/factory	1.100s
The fail isn't relevant to this PR.

@k8s-bot  test this please

**Pull request merged**